File: telegram-bot-bumblebee-core/src/main/java/com/github/bumblebee/command/autocomplete/entity/AutoCompletePhrase.java
package com.github.bumblebee.command.autocomplete.entity;

import javax.persistence.*;

@Entity
@Table(name = "AUTOCOMPLETE_PHRASES")
public class AutoCompletePhrase {

    @Id
    private String phraseKey;

    private String phrasePattern;

    public AutoCompletePhrase() {

    }

    public AutoCompletePhrase(String phraseKey, String phrasePattern) {
        this.phraseKey = phraseKey;
        this.phrasePattern = phrasePattern;
    }

    public void setPhraseToComplete(String phraseKey) {
        this.phraseKey = phraseKey;
    }

    public void setCompletePhrase(String phrasePattern) {
        this.phrasePattern = phrasePattern;
    }

    public String getPhraseKey() {
        return phraseKey;
    }

    public String getPhrasePattern() {
        return phrasePattern;
    }

    @Override
    public String toString() {
        return "PhraseKey:" + phraseKey
                + " PhrasePattern:" + phrasePattern;
    }
}

File: telegram-bot-bumblebee-core/src/main/kotlin/com/github/bumblebee/command/autocomplete/entity/AutoCompletePhrase.kt
package com.github.bumblebee.command.autocomplete.entity

import javax.persistence.Entity
import javax.persistence.Id
import javax.persistence.Table

@Entity
@Table(name = "BB_AUTOCOMPLETE_PHRASES")
class AutoCompletePhrase {

    @Id
    lateinit var phraseKey: String
    lateinit var phrasePattern: String

    @Suppress("unused")
    constructor()

    constructor(phraseKey: String, phrasePattern: String) {
        this.phraseKey = phraseKey
        this.phrasePattern = phrasePattern
    }

    override fun toString(): String = "AutoCompletePhrase(phraseKey=$phraseKey, phrasePattern=$phrasePattern)"
}
